acceptance-tests/helpers/apps/start.go

```diff
@@ -0,0 +1,21 @@
+package apps
+
+import (
+	"acceptancetests/helpers/cf"
+	"time"
+
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+)
+
+func (a App) Start() {
+	Start(a)
+}
+
+func Start(apps ...App) {
```

Comment:
Should this be a private function?

Reply:
No, because we often perform the same action on multiple apps.